### PR TITLE
update currentBroadcastStartTime when authentictae is called

### PR DESCRIPTION
**Pull request by yossipapi**

### CODE REVIEW

File: plugins/media/wowza/lib/model/WowzaMediaServerNode.php [hunk 1]
```diff
@@ -134,8 +134,8 @@ public function getApplicationPrefix($mediaServerConfig)
 
 		$appPrefix = $this->getValueByField($mediaServerConfig, 'appPrefix', $appPrefix);
 		
-		if($this->getAppPrefix())
-			$appPrefix = $this->getApplicationPrefix();
+		if(is_null($this->getAppPrefix()))
+			$appPrefix = $this->getAppPrefix();
```
Comment (EladCohenKaltura) [inline plugins/media/wowza/lib/model/WowzaMediaServerNode.php]: It seems that you will do this only in the case it is null - i think you want the opposite

Comment (yossipapi) [inline plugins/media/wowza/lib/model/WowzaMediaServerNode.php]: You are right my bad... The not somehow disappeared... Will add it and update the pull request.

Thanks for the heads up.

File: alpha/lib/model/LiveEntryServerNode.php [hunk 2]
```diff
@@ -44,9 +65,21 @@ public function postDelete(PropelPDO $con = null)
...
+			{
+				$liveEntry->setPrimaryServerNodeId(null);
+				
+				$entryServerNodes = EntryServerNodePeer::retrieveByEntryId($this->getEntryId());
+				if($liveEntry->getCurrentBroadcastStartTime())
+					$liveEntry->setCurrentBroadcastStartTime(0);
+			}
+			
+			if(!$liveEntry->save())
+				$liveEntry->indexToSearchIndex();
```
Comment (EladCohenKaltura) [inline alpha/lib/model/LiveEntryServerNode.php]: Save should generally work - I think that it is better to call $liveEntry->indexToSearchIndex(); within the if ($this->getServerType() === EntryServerNodeType::LIVE_PRIMARY) condition

**Comment by yossipapi** [comment]
@EladCohenKaltura added the ! before the is_null as commented... did you review the entire code or only this section?

**Pull request merged**